pmpro-gift-aid.php

```diff
@@ -22,6 +22,10 @@ function pmproga_pmpro_checkout_after_level_cost()
 	}
 	else
 		$gift_aid = false;
+
+	global $pmpro_level;

```

Comment:
```suggestion
	$pmpro_level = pmpro_getLevelAtCheckout();
```
Let's get the level fresh instead of trusting the global.